encoder/vaapiencoder_h264.cpp

```diff
@@ -1661,11 +1661,12 @@ bool VaapiEncoderH264::addPackedPrefixNalUnit(const PicturePtr& picture) const
 
     bit_writer_write_trailing_bits(&bs);
 
+    uint32_t codedBits = bs.getCodedBitsCount();
```

Comment:
could we have some comments on code, for the pad thing?  With time pass, not everybody will read the commit log. 
Do we have problem in hevc and jpeg? since it use addPackedHeader